src/OrchardCore.Modules/OrchardCore.Setup/Views/_Layout.cshtml

```diff
@@ -1,4 +1,18 @@
+@using Microsoft.AspNetCore.Builder
+@using Microsoft.AspNetCore.Localization
+@using Microsoft.Extensions.Options
+@using System.Globalization; 
+@inject Microsoft.AspNetCore.Mvc.Localization.IViewLocalizer T
 @inject OrchardCore.IOrchardHelper Orchard
+@inject IOptions<RequestLocalizationOptions> LocOptions
+
+@{
+    var requestCulture = Context.Features.Get<IRequestCultureFeature>();
+
+    var currentCulture = CultureInfo.GetCultureInfo(Orchard.HttpContext.Request.Query.ContainsKey("ui-culture")
```

Comment:
@Skrypt I'm not sure why we need the check again, while the `LocOptions.Value.DefaultRequestCulture.Culture.Name` will pick up the current culture automatically from `QueryString`

Reply:
No it's not. It takes the default culture that we set in the startup.cs file.

Reply:
Sorry, it's my bad I mean `requestCulture`

Reply:
Ok you are right I removed this double check